helm/polaris/templates/deployment.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "polaris.fullname" . }}
+  labels:
+    {{- include "polaris.labels" . | nindent 4 }}
+    {{- if .Values.podLabels }}
+    {{- tpl (toYaml .Values.podLabels) . | nindent 4 }}
+    {{- end }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "polaris.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        {{- if .Values.podAnnotations }}
+          {{- tpl (toYaml .Values.podAnnotations) . | nindent 8 }}
+        {{- end }}
+      labels:
+        {{- include "polaris.selectorLabels" . | nindent 8 }}
+        {{- if .Values.podLabels }}
+        {{- tpl (toYaml .Values.podLabels) . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- if .Values.persistenceConfigSecret }}
+      initContainers:
+        - name: init-config
+          image: "{{ tpl .Values.initImage.repository . }}:{{ tpl .Values.initImage.tag . }}"
+          imagePullPolicy: {{ tpl .Values.initImage.pullPolicy . }}
+          command: ['sh', '-c', 'cd /secret && jar -cf /tmp/conf.jar persistence.xml']
```

Comment:
nit: `jar -C /secret` can probably avoid the `sh` complications

Reply:
How is `conf.jar` actually used (when it is generated)? Sorry, I could not trace it down :sweat_smile:

Reply:
`conf.jar` is the only way as of now to overwrite the `persistenct.xml`. Details in https://github.com/polaris-catalog/polaris/pull/115

Reply:
Good point for the `-C` option for `jar` command. Updated and remove dependencies on `cd` and `sh`